src/controllers/UserRequests.js

```diff
@@ -35,12 +35,15 @@ export function sendApi() {
     //id: transcriptOutput, //variable id
     apiKey: submittedApiKey,
     apiVersion: submittedApiVersion,
+    emailAddress: submittedAddress,
+    password: submittedPassword,
+    diagramID: submittedDiagramID
     //???????
```

Comment:
This is old code where Martin and I were testing how to send the active tab to the backend and it can be removed to clean up the code.